src/test/java/gov/nih/nci/evs/api/controller/SearchControllerTests.java

```diff
@@ -3875,13 +3877,37 @@ public void testSparqlVariations() throws Exception {
     }
   }
 
+  @Test
+  public void testSparqlFailureVariations() throws Exception {
```

Comment:
Test names leads me to believe there are multiple paths for failure but only one is being tested. What failure is this testing exactly?

Reply:
@bcarlsenca